src/hooks/listHooks.js

```diff
@@ -64,6 +65,31 @@ export const useRenderItem = (query, navigation, options = {}) => {
       );
       break;
     }
+    case LIST_TYPES.TEXT_WITH_BIG_CARD_LIST: {
+      renderItem = ({ item, index }) => {
+        if (index === 0) {
+          return (
+            <BigCardListItem navigation={navigation} horizontal={options.horizontal} item={item} />
+          );
+        } else {
+          return (
+            <TextListItem
+              item={{
+                ...item,
+                bottomDivider: true
```

Comment:
is this ok to always `true`? compared to the existing text list item there is a logic for the bottom divider

Reply:
https://github.com/smart-village-solutions/smart-village-app-app/commit/74e7eb7f2b529145ccbb1f97a902aa1981ad5341